.github/workflows/check.yml

```diff
@@ -7,8 +7,6 @@ on:
     branches: ["main"]
     paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
   merge_group:
-    branches: ["main"]
-    paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
```

Comment:
Oh, I did not know. My bad.

I guess it is fine without, given that CI time isn't critical for GitHub Merge Queue runs?!

See also https://github.com/orgs/community/discussions/45899.

Reply:
I guess so. I was just getting annoyed at vscode complaining about the invalid components.